indico/modules/events/persons/templates/management/_person_list_row.html

```diff
@@ -21,61 +38,32 @@
     <td class="i-table affiliation-column">
         {{ person.affiliation }}
     </td>
-    <td class="i-table">
-        <span class="i-label {{ 'no-role' if not person_data.roles.chairperson }}">
-            {%- if person.event.type == 'lecture' -%}
-                {%- trans %}Speaker{% endtrans -%}
-            {%- else -%}
-                {%- trans %}Chairperson{% endtrans -%}
+    <td class="i-table roles-column">
+        {% for role, role_data in person_data.roles.iteritems() %}
+            {%- if role_data is mapping %}
+                {%- if 'elements' in role_data -%}
```

Comment:
unless you really need to know whether it exists (even if empty) or is really not present, you could just use `role_data.elements`